README.md

```diff
@@ -410,6 +411,18 @@ tests:
       column_type_list: [date, datetime]
 ```
 
+### [expect_column_values_to_have_consistent_casing](macros/schema_tests/column_values_basic/expect_column_values_to_have_consistent_casing.sql)
+
+Expect a column to have consistent casing. By setting display_inconsistent_columns to true, the inconsistent columns will be displayed.
```

Comment:
1) Small comment: should be `display_inconsistent_columns` (in single markdown backticks)
2) I think we should be more clear here - the columns will only be displayed if you run the compiled test SQL for debugging in your console, nothing will be displayed to the screen during testing right?

Reply:
1. Fine, will adjust.
2. In the console, the number of rows is gonna reflect the number of inconsistent values in the column. If you run the compiled test, you will get the results.